src/InsertionsClient.Console/ConsoleApp/Program.cs

```diff
@@ -120,9 +120,15 @@ private static int Main(string[] args)
 
             IInsertionApiFactory apiFactory = new InsertionApiFactory();
             IInsertionApi api = apiFactory.Create(MaxWaitDuration, MaxDownloadDuration, MaxConcurrency);
+            List<string> manifestFiles = InputLoading.LoadManifestPaths(ManifestFile, out int invalidManifestFileCount);
             IEnumerable<Regex> whitelistedPackages = InputLoading.LoadWhitelistedPackages(WhitelistedPackagesFile);
             ImmutableHashSet<string> ignoredPackages = ImmutableHashSet<string>.Empty;
 
+            if(invalidManifestFileCount != 0)
```

Comment:
Nit: we know the file count, would it be better to show count instead of 'one or more'. 
Also probably can be useful to show the path for invalid files.

Reply:
All the invalid paths are printed to the console in `LoadWhitelistedPackages` method (when the actual parsing happens) before we execute this line. This check is here just to exit the app.

Reply:
sorry, actually selected wrong line, the comment is about line 129. Error message is `Failed to find one or more`, he suggestion is to show exact count as we know file count. 

ok for path.